### ISSUE TITLE
WSL fails to build depends if WSL appends Windows path

### ISSUE THREAD

**Issue by christiancfifi** [dashpay/dash#4376]
### Describe the issue
Error while building depends on any branch,  persisting through a reinstall of WSL and Ubuntu:
```/bin/sh: 1: Syntax error: "(" unexpected```
Solved by configuring WSL to not append the Windows path to the Ubuntu path.  See bitcoin/bitcoin#10269. Maybe the latest WSL changed the path behaviour. Instead of the proposed fix on bitcoin a much cleaner approach is to add this to /etc/wsl.conf
```
[interop]
appendWindowsPath=false
```

### Can you reliably reproduce the issue?
Yes
#### If so, please list the steps to reproduce below:
1. Reinstall WSL and Ubuntu
2. Attempt to build any dash branch

### Machine specs:
- OS: Windows 10 Build 19042

**Comment by christiancfifi** [comment]
It looks like this is already accounted for when cross-compiling for windows from WSL, but not when building for linux from WSL, which isn't a supported way to build (although outside of this issue building for linux from wsl has been working just fine). So, you can close this or I can add the fix to the windows build docs if you want to keep support for linux builds on WSL.

**Comment by PastaPastaPasta** [comment]
I think if you'd like to create a PR adding a bit more info to the documentation, we'd be happy to consider it.

**Comment by kittywhiskers** [comment]
This was already addressed in https://github.com/dashpay/dash/pull/2939 but our Windows documentation is lagging behind (that's the thing with build systems, even if your codebase is older, your build system cannot be :P) 

We _could_ just backport https://github.com/bitcoin/bitcoin/pull/11437 but read further and you will find... 

> Currently only building on Ubuntu Trusty 14.04 or Ubuntu Zesty 17.04 or later is supported.

...and now you see why backporting that alone is a bad idea

TL;DR: Windows documentation update is overdue, thanks for reporting!
